src/tini.c

```diff
@@ -50,17 +50,51 @@ typedef struct {
    struct sigaction* const sigttou_action_ptr;
 } signal_configuration_t;
 
+static char* const signal_names[] = {
+   "SIGHUP",
+   "SIGINT",
+   "SIGQUIT",
+   "SIGILL",
+   "SIGTRAP",
+   "SIGABRT",
+   "SIGBUS",
+   "SIGFPE",
+   "SIGKILL",
+   "SIGUSR1",
+   "SIGSEGV",
+   "SIGUSR2",
+   "SIGPIPE",
+   "SIGALRM",
+   "SIGTERM",
+   "SIGSTKFLT",
+   "SIGCHLD",
+   "SIGCONT",
+   "SIGSTOP",
+   "SIGTSTP",
+   "SIGTTIN",
+   "SIGTTOU",
+   "SIGURG",
+   "SIGXCPU",
+   "SIGXFSZ",
+   "SIGVTALRM",
+   "SIGPROF",
+   "SIGWINCH",
+   "SIGIO",
+   "SIGPWR",
+   "SIGSYS",
```

Comment:
It's a little more verbose... but I'd very much prefer having an array that doesn't actually depend on ordering to be safer. In other words, I'd rather have this be an array of [signame, signum] pairs, rather than have the signum be implicitly provided by the string's position in the array.